reporter/src/main/kotlin/reporters/AntennaAttributionDocumentReporter.kt

```diff
@@ -132,36 +107,54 @@ class AntennaAttributionDocumentReporter : Reporter {
             }
         }
 
-        val generator = AttributionDocumentGeneratorImpl(
-            reportFilename,
-            workingDir,
-            templateId,
-            DocumentValues(rootProject.id.name, rootProject.id.version, projectCopyright)
-        )
+        val projects = input.ortResult.getProjects(omitExcluded = true)
+        val outputFiles = mutableListOf<File>()
+        try {
+            projects.forEach { project ->
+                val dependencies = project.collectDependencies()
+
+                val packages = input.ortResult.getPackages().mapNotNull { (pkg, _) ->
+                    pkg.takeIf { dependencies.contains(pkg.id) }
```

Comment:
You can write this as `pkg.takeIf { it.id in dependencies }` (`in` is an infix operator for `contains()`).